browser/brave_content_browser_client.cc

```diff
@@ -846,6 +846,17 @@ bool BraveContentBrowserClient::HandleURLOverrideRewrite(
     return true;
   }
 
+#if !BUILDFLAG(IS_ANDROID)
```

Comment:
@simonhong i've added this re-write and it should be for desktop only

Reply:
It would be great to have test case for this rewriting.

Reply:
added